### DISCUSSION TITLE
How to get the coordinate information of polygons drawn in bounding box in object dectection problem

### DISCUSSION THREAD

**Discussion by dinhkhoi12tin** [facebookresearch/detectron2#3846] [Q&A]
Hello everyone,

In Detectron2 how to get the polygon bounding box around object not a rectangle box
I already have seen tutorial , it's just a rectangle box , it's mean format box is x,y,h,w .
So What I want is format 4 points x1,y1,x2,y2,x3,y3,x4,y4 

How can ?

many thanks

**Comment by MathijsNL** [comment]
You can calculate the points yourself.

https://stackoverflow.com/questions/56115874/how-to-convert-bounding-box-x1-y1-x2-y2-to-yolo-style-x-y-w-h/56121386#56121386

Guess you just need to reverse that since that question is the other way around.

**Comment by dinhkhoi12tin** [comment]
> You can calculate the points yourself.
> 
> https://stackoverflow.com/questions/56115874/how-to-convert-bounding-box-x1-y1-x2-y2-to-yolo-style-x-y-w-h/56121386#56121386
> 
> Guess you just need to reverse that since that question is the other way around.

First Thanks for reply me.
But you confused my question. My question is How to get a polygons Which I mean is quadrilateral not rectangle box

**Comment by MathijsNL** [comment]
Do you have an example image to illustrate what you want?
Also there is a lot you can do by using the bitmask in case you get one as part of the result.
